templates/template-accounts-payable-invoice-processing/.env.example

```
@@ -0,0 +1,36 @@
+# Fixture workflows need no external credentials; Studio attachment processing uses OPENAI_API_KEY.
+INVOICE_READER=fixture
+INVOICE_READER_MODEL=openai/gpt-5.2
+OPENAI_API_KEY=
+INVOICE_ROOT=.
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The INVOICE_ROOT key should go before the OPENAI_API_KEY key

(UnorderedKey)